astropy/units/format/generic_parsetab.py

```diff
@@ -1,77 +1,76 @@
-from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
This line needs to stay (even though PLY doesn't generate it) so that the tokens are unicode.